File: C/10.9/guess.c
//Paola Frunzio
//10.9.2020
//user keeps entering number until they guess your number. tell them high or low

#include <stdio.h>
#include <cs50.h>

int main(void)
{
    int guess = 0;
    int num = 56;
    guess = get_int("Number: ");
    
    while (guess != num)
    {
        if (guess < num)
        {
            printf("Higher\n");
        }
        else
        {
            printf("Lower\n");
        }
        guess = get_int("Number: ");
    }
    printf("You got it!");
}

File: Python/1.8/guess.py
# Paola Frunzio
# 1.8.2021
# guess number

from cs50 import get_int
import random

num = random.randint(0,1000)
guess = 0

while(guess != num):
    guess = get_int("Guess the number: ")
    if guess > num:
        print("lower")
    elif guess < num:
        print("higher")

print("yup")